HW#1/script.js

```diff
@@ -0,0 +1,16 @@
+'use strict'
+
+const firstNumber = 10;
+const secondNumber = 5;
+
+let addition = (firstNumber + secondNumber);
```

Comment:
Ця змінна (і наступні) не перезаписується, тому її варто об'являти через константу. І в цьому випадку обгортати математичну операцію в дужки нема потреби.
```suggestion
const addition = firstNumber + secondNumber;
```